cves/CVE-2015-6768.yml

```
@@ -32,7 +32,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: When stopping an HTML frame and its children, there is a point where
```

It seems as though your build is failing. Multi-line answers should be like my suggestion below. it should be like this for any multi-line answers in both your files. Please fix them and make sure the build passes. For another example, look how the multi-line instructions handle it.
```suggestion
description: |
  When stopping an HTML frame and its children, there is a point where
```